File: src/main/java/ru/dragonestia/apocalypse/item/TinNuggetItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class TinNuggetItem extends Item implements AdvItem {

    public TinNuggetItem(Integer meta, int count) {
        super(ApocalypseID.TIN_NUGGET, meta, count, "Tin nugget");
    }

    @Override
    public String getRuName() {
        return "Оловянный самородок";
    }

    @Override
    public String getIcon() {
        return "textures/items/nugget/tin";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/TinNuggetItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class TinNuggetItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.TIN_NUGGET, 0, count, "Tin nugget"), AdvItem {

    override fun getRuName(): String = "Оловянный самородок"

    override fun getIcon(): String = "textures/items/nugget/tin"

}